src/debug/di/rsthread.cpp

```diff
@@ -8915,8 +8915,11 @@ HRESULT CordbJITILFrame::GetReturnValueForILOffsetImpl(ULONG32 ILoffset, ICorDeb
     bool found = false;
     ULONG32 currentOffset = m_nativeFrame->GetIPOffset();
     for (ULONG32 i = 0; i < count; ++i)
```

Comment:
Could you please add extra scope here so that the formatting is consistent with the rest of the file?